ops/terraform/modules/aou-rw-reporting/schemas/cohort.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "name": "cohort_id",
-    "type": "INT64",
+    "type": "INTEGER",
```

Comment:
INT64 gets converted to INTEGER by BQ, and then Terraform perpetually thinks it needs to change it back. So if you can't beat 'em...